solr/solrj/src/test/org/apache/solr/client/solrj/request/TestUpdateRequestCodec.java

```diff
@@ -247,13 +247,14 @@ public void testBackCompat4_5() throws IOException {
 
     InputStream is = getClass().getResourceAsStream("/solrj/updateReq_4_5.bin");
     assertNotNull("updateReq_4_5.bin was not found", is);
+    List<SolrInputDocument> unmarshalledDocs = new ArrayList<>();
     UpdateRequest updateUnmarshalled =
         new JavaBinUpdateRequestCodec()
             .unmarshal(
                 is,
                 (document, req, commitWithin, override) -> {
                   if (commitWithin == null) {
-                    req.add(document);
```

Comment:
The test needed an update because JavaBinUpdateRequestCodec no longer uses the same UpdateRequest between the add-docs and delete-docs parts, which doesn't matter.  Arguably, the callback shouldn't even take an UpdateRequest anyway.